hooks/post_gen_project.py

```diff
@@ -26,19 +26,31 @@ def invoke_shell(command):
         print(e.output)
         raise e
     print(decode_string(output))
+    if return_output:
+            return decode_string(output)
```

Comment:
Indention looks off here, might as well *always* return this value.

Reply:
I agree that we may as well always return this.